src/proto/Cosmos.proto

```diff
@@ -119,6 +119,14 @@ message Message {
         repeated string coin = 6;
     }
 
+    // thorchain/MsgSend
+    message THORChainSend {
+        bytes from_address = 1;
```

Comment:
Let's use string like other messages?

Reply:
In the proto the bytes address contains the key hash, not the Bech32 string version.  Therefore I would keep that here as well; string is not right for binary keyhash.